CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ Release built: _not published yet_
 
 ### API Changes
 - Changed `variant_id` of `ProgrammaticScryptoSborValueEnum` from numeric (`type: integer`) to string-encoded numeric (`type: string`) to make it compatible with the rest of the ecosystem.
+- Added `unix_timestamp_seconds` property to `MetadataInstantValue` and its array counterpart with exact numerical timestamp value while `value` retains RFC-3339 compatible date-time clamped to 1583..9999 year range.
```

Comment:
A few suggested tweaks to order, to put the more important clamp quasi-breaking change first, and then mention the new property.
And also added some missing articles. (You can either say " `unix_timestamp_seconds` " or "a `unix_timestamp_seconds` property" but not "`unix_timestamp_seconds` property"). The second one sounds a little more formal, so good for release notes and tech docs.
```suggestion
- Changed the `MetadataInstantValue` type and its array counterpart `MetadataInstantArrayValue` to clamp the `value` property within the RFC-3339 compatible date-time year range `1583` to `9999`. Added a `unix_timestamp_seconds` property to these types to give the exact unclamped numerical timestamp value. 
```